clients/include/utility.hpp

```diff
@@ -90,7 +90,7 @@ class rocblas_local_handle
     {
         return handle;
     }
-    operator const rocblas_handle&() const
```

Comment:
Not sure if clang format is messed up or this is now preferred by someone.

Reply:
Had a random no-npi build from bug-hunting awhile back which I guess has a different clang-format. Upgraded to 3.1 to match the node this was building on and it's better now.